### ISSUE TITLE
Login Screen - Screen Reader does not read titles / instructions

### ISSUE THREAD

**Issue by jeromeleecms** [18F/cms-hitech-apd#1301]
Screen reader is not reading content / not accessible
- Instructions for "Please log in." (also not keyboard accessible)

### This task is done when...
 - [ ] Screen Reader can access and read "Please log in." (or other instructions for login page)

**Comment by jeromeleecms** [comment]
@nkkl could you please take a look to see if this in the ballpark for what is expected for testing? We can close this out and/or move the content into a general Google doc for review once the design system + table changes have been made (or we can keep this pending and remove the assignment).

**Comment by nkkl** [comment]
Looks like this was fixed! :)

**Comment by nkkl** [comment]
Or at least it works correctly with VoiceOver.

**Comment by jeromeleecms** [comment]
I've made some progress on getting Windows Narrator checked out/fixed. I'll close this for now and retest everything again once I can do a more complete review of the app with it working.
